File: PReflection/PReflection/Categoria.cs
using System;

namespace PReflection
{
	public class Categoria
	{	

		public ulong Id { get; set;}
		[NotBlankAttribute]
		public string  Nombre { get; set;}
		public Categoria (ulong id, string nombre)
		{
			this.Id = id;
			this.Nombre = nombre;
		}
	}
}

File: CategoriaGradle/src/main/java/Categoria.java
import javax.persistence.Entity;
import javax.persistence.Id;


@Entity
public class Categoria {
	
	//@Table( name = "Categoria" )
	public class Event {
	   private Long id;
	   private String nombre;
	}
	
	@Id
	public Long getId(){
		return id;
	}
	
	private void setId(Long Id){
		this.id = id;
	}
}
